components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Create/CreateSteps/Step1.jsx

```diff
@@ -131,47 +132,37 @@ class Step1 extends Component {
                 <div>
                     <div>
                         <FormGroup>
-                            <Label for="app-title">Title*</Label>
-                            <Input
-                                required
-                                type="text"
-                                name="appName"
-                                id="app-title"
-                            />
+                            <Label for="app-title">
+                                <FormattedMessage id='Title' defaultMessage='Title'/>*
+                            </Label>
+                            <Input required type="text" name="appName" id="app-title"/>
                         </FormGroup>
                         <FormGroup>
-                            <Label for="app-description">Description*</Label>
-                            <Input
-                                required
-                                type="textarea"
-                                name="appDescription"
-                                id="app-description"
-                            />
+                            <Label for="app-description">
+                                <FormattedMessage id='Description' defaultMessage='Description'/>*
+                            </Label>
+                            <Input required type="textarea" name="appDescription" id="app-description"/>
                         </FormGroup>
                         <FormGroup>
-                            <Label for="app-category">Category</Label>
-                            <Input
-                                type="select"
-                                name="category"
-                                id="app-category"
-                            >
+                            <Label for="app-category">
+                                <FormattedMessage id='Category' defaultMessage='Category'/>
+                            </Label>
+                            <Input type="select" name="category" id="app-category">
                                 <option>Business</option>
```

Comment:
Isn't this shouldn't be used by localization?

Reply:
This is just a sample value. These should be actually taken back-end. So we do not need to add the localization here.